.github/CODEOWNERS

```diff
@@ -1,124 +1,294 @@
+# Secondary maintainer @herbelin
+
+/theories/Classes/        @mattam82
+# Secondary maintainer @herbelin
+
+/theories/FSets/          @letouzey
+# Secondary maintainer @herbelin
+
+/theories/Init/           @letouzey
+# Secondary maintainer @ppedrot
+
+/theories/Lists/          @letouzey
+# Secondary maintainer @ppedrot
+
+/theories/Logic/          @herbelin
+# Secondary maintainer @ppedrot
+
+/theories/MSets/          @letouzey
+# Secondary maintainer @herbelin
+
+/theories/NArith/         @letouzey
+# Secondary maintainer @herbelin
+
+/theories/Numbers/        @letouzey
+# Secondary maintainer @herbelin
+
+/theories/PArith/         @letouzey
+# Secondary maintainer @herbelin
+
+/theories/Program/        @mattam82
+# Secondary maintainer @herbelin
+
+/theories/QArith/         @letouzey
+# Secondary maintainer @herbelin
+
+/theories/Reals/          @silene
+# Secondary maintainer @ppedrot
+
+/theories/Relations/      @mattam82
+# Secondary maintainer @ppedrot
+
+/theories/Setoids/        @mattam82
+# Secondary maintainer @ppedrot
+
+/theories/Sets/           @letouzey
+# Secondary maintainer @herbelin
+
+/theories/Sorting/        @letouzey
+# Secondary maintainer @herbelin
+
+/theories/Strings/        @letouzey
+# Secondary maintainer @herbelin
+
+/theories/Structures/     @letouzey
+# Secondary maintainer @herbelin
+
+/theories/Unicode/        @letouzey
+# Secondary maintainer @herbelin
+
+/theories/Wellfounded/    @letouzey
+# Secondary maintainer @mattam82
+
+/theories/ZArith/         @letouzey
+# Secondary maintainer @herbelin
+
+/theories/Compat/         @JasonGross
+# Secondary maintainer @Zimmi48
 
 /theories/Vectors/        @herbelin
 
-/tools/coqdoc/          @silene @mattam82
-/tools/coq_makefile*    @gares @silene
-/tools/CoqMakefile*     @gares @silene
-/tools/coqdep*          @ppedrot @maximedenes
-/tools/coq_tex*         @silene @gares
-/tools/coqwc*           @silene @gares
+########## Tools ##########
+
+/tools/coqdoc/          @silene
+# Secondary maintainer @mattam82
+
+/tools/coq_makefile*    @gares
+# Secondary maintainer @silene
+
+/tools/CoqMakefile*     @gares
+# Secondary maintainer @silene
+
+/tools/coqdep*          @ppedrot
+# Secondary maintainer @maximedenes
+
+/tools/coq_tex*         @silene
+# Secondary maintainer @gares
+
+/tools/coqwc*           @silene
+# Secondary maintainer @gares
+
+########## Toplevel ##########
+
+/toplevel/        @ejgallego
+# Secondary maintainer @gares
+
+########## Vernacular ##########
+
+/vernac/          @mattam82
+# Secondary maintainer @maximedenes
+
+########## Build system ##########
+
+/Makefile*        @letouzey
+# Secondary maintainer @maximdenes
 
-/toplevel/        @ejgallego @gares
-/vernac/          @mattam82 @maximedenes
+/configure*       @letouzey
+# Secondary maintainer @ejgallego
 
-/Makefile*        @letouzey @ejgallego
-/configure*       @letouzey @ejgallego
-/META.coq         @letouzey @ejgallego
+/META.coq         @letouzey
+# Secondary maintainer @ejgallego
```

Comment:
Why isn't @ejgallego the primary for META.coq?

Reply:
Right, seems more logical. Fixed.